packages/http-server-javascript/build-helpers.ts

```diff
@@ -0,0 +1,167 @@
+/* eslint-disable */
+
+import fs from "node:fs/promises";
+import { EOL } from "node:os";
+import path from "node:path";
+
+const HELPER_DECLARATION_PATH = path.resolve([".", "src", "helpers"].join(path.sep));
```

Comment:
```suggestion
const HELPER_DECLARATION_PATH = path.resolve(".", "src", "helpers");
```

I think here you can just do that no?